### ISSUE TITLE
PlatformIO support

### ISSUE THREAD

**Issue by Andy79881** [SpenceKonde/megaTinyCore#1104]
I would like to use VScode with platformIO instead of Arduino IDE. Is MegaTinyCore supported? Especially I need to check a certain option (using TCA0 for millis() ). I could not figure out how to do that. Do you have any hints to point me in the right direction? Thanks in advance

**Comment by T94T** [comment]
See: https://github.com/SpenceKonde/megaTinyCore/blob/master/megaavr/extras/PlatformIO.md

You need to set `platform = atmelmegaavr`

**Comment by SpenceKonde** [comment]
Yeah, please see that document. And if you have further questions, I would suggest asking a person who has  a high liklihood of knowing the answer (which rules me out)

**Comment by SpenceKonde** [comment]
Closing this as not actionable. Please report any issues relating to PIO with the understanding that I can fix a problem if you tell me exactly what is wrong that is causing PIO issues, but I cannot debug, or write documentation or give guidance for PIO users because I don't use PIO and do not have the time to take on learning a new build environment. I hate build environments and
